website/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Vella</title>
+    <style>
+        html {
+            min-height: 100vh;
+            display: grid;
+        }
+
+        * {
+            box-sizing: border-box;
+        }
+
+        body {
+            margin: 0px;
+            background: rgb(63, 94, 251);
+            background: linear-gradient(0deg, rgb(255, 57, 248) 0%, rgb(255, 179, 134) 100%);
+            color: white;
+            display: grid;
+            justify-content: center;
+            justify-items: center;
+            align-content: center;
+            gap: 2em;
+            font-family: Arial, Helvetica, sans-serif;
+            font-weight: 100;
+            padding: 1em;
+            height: 100%;
+            text-align: center;
+        }
+
+        h1 {
+            font-style: italic;
+            letter-spacing: 0.1em;
+            font-size: 10em;
+            margin: 0em;
+        }
+
+        a,
+        a:visited {
+            color: currentColor;
+        }
+
+        .logo {
+            border-radius: 100%;
+            max-width: 100%;
+            width: 8em;
+        }
+
+        .links {
+            display: grid;
+            grid-auto-flow: column;
+            justify-content: space-between;
+            gap: 1em;
+            text-transform: lowercase;
+            font-size: 2em;
+        }
+    </style>
+</head>
+
+<body>
+
+</body>
+
+</html>
+<h1>vella</h1>
+<p style="font-size: 2em">Deadly serious web design.</p>
+<div>
+</div>
+<div class="links">
+    <a href="https://github.com/dom-dee-dom/vella">Github</a>
+    <a href="https://discord.gg/R4Uv5Dv">Chat</a>
+    <a href="https://github.com/dom-dee-dom/vella/blob/master/README.md">Docs</a>
+</div>
+
+</html>
```

Comment:
A bit of a joke website to serve as an example on how to deploy and also as a starting point for someone to do an actual initial design for the docs.